doc/rbd/libvirt.rst

```diff
@@ -223,29 +223,27 @@ commands, refer to `Virsh Command Reference`_.
 	
 #. Save the file.
 
-#. If your Ceph Storage Cluster has `Ceph Authentication`_ enabled (it does by 
-   default), you must generate a secret. :: 
-
-	cat > secret.xml <<EOF
-	<secret ephemeral='no' private='no'>
-		<usage type='ceph'>
-			<name>client.libvirt secret</name>
-		</usage>
-	</secret>
-	EOF
-
-#. Define the secret. ::
-
-	sudo virsh secret-define --file secret.xml
-	<uuid of secret is output here>
-
 #. Get the ``client.libvirt`` key and save the key string to a file. ::
 
 	ceph auth get-key client.libvirt | sudo tee client.libvirt.key
 
-#. Set the UUID of the secret. :: 
+#. If your Ceph Storage Cluster has `Ceph Authentication`_ enabled (it does by
+   default), you must generate a secret. ::
 
-	sudo virsh secret-set-value --secret {uuid of secret} --base64 $(cat client.libvirt.key) && rm client.libvirt.key secret.xml
+    uuidgen
+    9ec59067-fdbc-a6c0-03ff-df165c0587b8
+
+    cat > secret.xml <<EOF
+    <secret ephemeral='no' private='no'>
+        <uuid>9ec59067-fdbc-a6c0-03ff-df165c0587b8<uuid/>
+        <usage type='ceph'>
+            <name>client.libvirt secret</name>
+        </usage>
+    </secret>
+    EOF
+    sudo virsh secret-define --file secret.xml
+    Secret 9ec59067-fdbc-a6c0-03ff-df165c0587b8 created
```

Comment:
Nit: it may be clearer to keep these steps separate. Right now, it's hard to differentiate between command-line inputs and outputs (like this line). Alternatively, perhaps prefix all CLI commands throughput this file w/ "$" or "#" to indicate a prompt.

Reply:
@dillaman It seems that we don't need to use "virsh secret-define" to get the uuid since it has been got through the command "uuid"
I want to align this document with the rbd-openstack document to avoid confusion.

Agree with adding prefix "$" or "#" in CLI commands. I'll refine it.

Reply:
Keep little change to refine the doc